src/components/text/pages/text.jsx

```diff
@@ -7,68 +7,119 @@ import {TEXT_TYPE, TEXT_SIZE, TEXT_COLOR, TEXT_WEIGHT, TEXT_TRANSFORM, TEXT_ALIG
 
 const text = 'We\'ve got your back!';
 
+const textSizesMap = [
+  {
+    type: 'xsmall',
+    fontSize: '12px'
+  },
+  {
+    type: 'small',
+    fontSize: '15px'
+  },
+  {
+    type: 'normal',
+    fontSize: '18px'
+  },
+  {
+    type: 'large',
+    fontSize: '24px'
+  },
+  {
+    type: 'xlarge',
+    fontSize: '33px'
+  },
+  {
+    type: 'xxlarge',
+    fontSize: '45px'
+  }
+];
+
 function getValues(object, addUndefined = true) {
   return addUndefined ? [undefined, ...Object.values(object)] : Object.values(object);
 }
 
 const TextExamples = () => {
   const SizeVariant = [];
-  const colorVariant = [];
+  const ColorVariant = [];
```

Comment:
`ColorVariant` => `colorVariant` ?
or is there any reason to start with capital letter?

Reply:
@matzimowski no reason, I am a wierdoo 🙈

Reply:
@matzimowski on my way to fix it.